wordle-web/components/game-board.vue

```diff
@@ -16,9 +20,9 @@
 
 <script lang="ts">
 import { Component, Vue, Prop } from 'vue-property-decorator'
-import { WordleGame } from '~/scripts/wordleGame'
-import { Word } from '~/scripts/word'
-import { Letter } from '~/scripts/letter'
+import { WordleGame } from '../scripts/wordleGame'
+import { Word } from '../scripts/word'
+import { Letter } from '../scripts/letter'
```

Comment:
Typically it's best practice to use the alias `@`  or `~` for the root directory defined in your `tsconfig` rather than doing file navigation with the `..` syntax.

The reason being, is that in larger projects. Lets say you have a component you wish to use a few directories up, you'd end up with imports that look like this:
`../../../dialogs/login/signin-dialog.vue` rather than something like `@/components/dialogs/login/sigin-dialog.vue` which looks cleaner, and if you change the directory of the .vue file those imports are in, they won't all break.